pages/createRoute.js

```diff
@@ -0,0 +1,30 @@
+import styled from "styled-components";
+import Header from "../components/Header";
+import BackButton from "../components/BackButton/index.js";
+import RouteForm from "../components/RouteForm";
+import { useRouter } from "next/router";
+import { useState } from "react";
+
+export default function CreateRoutePage() {
+    const [myRoutes, setMyRoutes] = useState([]);
+    const router = useRouter();
+  
+    function handleRouteCreated(newRoute) {
+      setMyRoutes((prevRoutes) => [...prevRoutes, newRoute]);
+      router.push("/myRoutes");
+    }
+return (
+    <div>
```

Comment:
I guess you dont need the div here. 
use <> instead.

Reply:
Done